pkg/appmgmt/appmgmt.go

```diff
@@ -56,21 +56,14 @@ func NewAMService(amProtocol interfaces.ApplicationManagementProtocol,
 	}
 
 	log.Logger().Info("Initializing new AppMgmt service")
-
-	if !apiProvider.IsTestingMode() {
-		log.Logger().Info("Registering Spark operator with the AppMgmt service")
```

Comment:
log.Logger().Info("Registering Spark operator with the AppMgmt service")
If this log don't need for new code?

Reply:
That line does not add too much information, we always register the Spark operator.